content/departments/product-engineering/product/design/workshops/async-design.md

```diff
@@ -0,0 +1,61 @@
+# Async design workshop
+
+If you want to avoid sync meetings and want to have a design workshop activity that can involve a large group of people you can consider an async design workshop.
+
+The activities you do on a given day are flexible with this approach, so you can do problem validation, solution validation, brainstorming, and more; the important part is making it a multi-day activity with offsetting days for content creation and evaluation/voting, with a date boundary in between.
+
+Benefits:
+
+- Supports async work
+- More flexibility when people are in lots of time zones
+- Can support quite large groups
+- Self-documenting
+- Allows time for reflection and "shower thoughts" to be included
+
+Drawbacks:
+
+- Plays out slower than if everyone was in a live session together
+- Not as engaging for people who are inspired by real-time communication
+- Requires lots of written communication
+
+## Scheduling considerations
+
+Because this is an activity that takes a little bit of attention each day over a few days or a week, its' good to avoid scheduling at the same time as the release is wrapping up, or another heavyweight organizational activity like planning is going on.
```

Comment:
```suggestion
Because this is an activity that takes a little bit of attention each day over a few days or a week, it's good to avoid scheduling at the same time as the release is wrapping up, or another heavyweight organizational activity like planning is going on.
```